docs/machine-learning/how-to-guides/serve-model-web-api-ml-net.md

```diff
@@ -0,0 +1,123 @@
+1. In Solution Explorer, right-click the model zip file and select Properties. Under Advanced, change the value of Copy to Output Directory to Copy if newer.
+
+## Build Data Models
+
+You need to create some classes for your input data and predictions. Add a new class to your project:
+
+1. Create a directory named Models in your project to save your data models:
+
+    In Solution Explorer, right-click on your project and select Add > New Folder. Type "Models" and hit Enter.
+
+2. In Solution Explorer, right-click the *Models* directory, and then select Add > New Item.
+3. In the Add New Item dialog box, select Class and change the Name field to SentimentData.cs. Then, select the Add button. The SentimentData.cs file opens in the code editor. Add the following using statement to the top of SentimentData.cs:
+
+[!code-csharp[AddUsings](../../../samples/machine-learning/how-tos/SentimentAnalysisWebAPI/Models/SentimentData.cs#1 "Add necessary usings")]
+
+Remove the existing class definition and add the following code to the SentimentData.cs file:
+
+[!code-csharp[SentimentClassDefinition](../../../samples/machine-learning/how-tos/SentimentAnalysisWebAPI/Models/SentimentData.cs#2 "Define Sentiment Data Class")]
+
+4. In Solution Explorer, right-click the *Models* directory, and then select Add > New Item.
+5. In the Add New Item dialog box, select Class and change the Name field to SentimentPrediction.cs. Then, select the Add button. The SentimentPrediction.cs file opens in the code editor. Add the following using statement to the top of SentimentPrediction.cs:
+
+[!code-csharp[AddUsings](../../../samples/machine-learning/how-tos/SentimentAnalysisWebAPI/Models/SentimentPrediction.cs#1 "Add necessary usings")]
+
+Remove the existing class definition and add the following code to the SentimentPrediction.cs file:
+
+[!code-csharp[SentimentPredictionDefinition](../../../samples/machine-learning/how-tos/SentimentAnalysisWebAPI/Models/SentimentPrediction.cs#1 "Define Sentiment Prediction Class")]
+
+## Create Prediction Service
+
+To organize and re-use the prediction logic throughout the entire application, create a prediction service.
+
+1. Create a directory named Services in your project to hold services to be used by the application:
+
+    In Solution Explorer, right-click on your project and select Add > New Folder. Type "Services" and hit Enter.
+
+1. In Solution Explorer, right-click the *Services* directory, and then select Add > New Item.
+1. In the Add New Item dialog box, select Class and change the Name field to PredictionService.cs. Then, select the Add button. The PredictionService.cs file opens in the code editor. Add the following using statement to the top of PredictionService.cs:
+
+[!code-csharp[AddUsings](../../../samples/machine-learning/how-tos/SentimentAnalysisWebAPI/Services/PredictionService.cs#1 "Add necessary usings")]
+
+Remove the existing class definition and add the following code to the PredictionService.cs file:
```

Comment:
Can you point me to a full URL for that prediction Service code so I can review it? :)

Reply:
https://github.com/luisquintanilla/SentimentAnalysisWebAPI